owncloudApp/src/main/java/com/owncloud/android/authentication/oauth/OAuthUtils.kt

```diff
@@ -58,7 +61,7 @@ class OAuthUtils {
             AuthorizationServiceConfiguration.fetchFromUrl(
                 serviceDiscoveryLocation,
                 onGetAuthorizationServiceConfiguration,
-                OAuthConnectionBuilder(context)
+                OkConnectionBuilder(HttpClient.getOkHttpClient())
```

Comment:
I see that you have replaced the old OAuthConnectionBuilder(context) here, and introduced the new one. But OAuthConnectionBuilder is used in some other places. Just curiosity, why do you replace it only here? By the way, this is the only appauth call that was logged. It may be related 🤔